### PR TITLE
feat: credo ci

### PR DESCRIPTION
**Pull request by kyle-annen**
Runs static code analysis in CI.

### CODE REVIEW

File: lib/chess_club_web/resolvers/scenario_resolver.ex [hunk 1]
```diff
@@ -1,34 +1,57 @@
...
+      Scenario
+      |> ChessClub.Repo.get(args.scenario_id)
+      |> ChessClub.Repo.preload(:moves)
+      |> enrich_scenario()
+
+    {:ok, scenario_with_state_and_moves}
   end
 
-  defp resolve_scenario(scenario) do
+  defp enrich_scenario(scenario) do
```
Comment (7hoenix) [inline lib/chess_club_web/resolvers/scenario_resolver.ex:54]: Love this name choice.

Comment (kyle-annen) [reply lib/chess_club_web/resolvers/scenario_resolver.ex:54]: 👍

**Comment by 7hoenix** [comment]
This is awesome 🎉 🎉 🎉 . :shipit:

**Comment by kyle-annen** [comment]
Closing due to strange CI state not getting triggered on new commit.

**Pull request closed without merging**